common/src/main/java/fi/hel/verkkokauppa/common/queue/error/DefaultActiveMQErrorHandler.java

```diff
@@ -10,9 +12,21 @@
 @Slf4j
 public class DefaultActiveMQErrorHandler implements ErrorHandler {
 
+    private ObjectMapper mapper;
+
+    public DefaultActiveMQErrorHandler(ObjectMapper mapper) {
+        this.mapper = mapper;
+    }
+
     @Override
     public void handleError(Throwable t) {
-        log.error(t.getCause().getMessage());
+        try {
+            log.error(mapper.writeValueAsString(t));
```

Comment:
add prefix for this log, like

log.error("DefaultActiveMqError: "+ mapper.writeValueAsString(t))